### ISSUE TITLE
Administration : Form title missing language attribute

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2384]
The form title is broken into English and French; the French should have a lang="fr" attribute while using the English interface and vice versa.

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/013a31ed-b86f-4ddf-9c70-3b0f18a4e25c)

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/721a43b6-b33b-4e85-b52d-6a98d71bf6cc)

**Comment by timarney** [comment]
Merged https://github.com/cds-snc/platform-forms-client/pull/2388
